src/makielayout_integration.jl

```diff
@@ -92,6 +105,72 @@ function layoutplot!(scene, layout, ts::Algebraic)
             @warn "Automated legend was not possible due to $e"
         end
     end
+    
+    ax1 = axs[end,1]
+    
+    # faceting: hide x and y labels
+    for i in 1:length(facetlayout.content)
+        ax = facetlayout.content[i].content
+        ax.xlabelvisible[] = ax.xlabelvisible[] && !has_layout_x(ts)
+        ax.ylabelvisible[] = ax.ylabelvisible[] && !has_layout_y(ts)
+    end
+        
+
+    if has_layout_x(ts)
+        # Facet labels
+        lxl = string.(layout_x_levels(ts))
+        @assert length(lxl) == Nx
+        for i in 1:Nx
+            text = LText(scene, lxl[i])
+            facetlayout[1, i, Top()] = LRect(
+                scene, color = RGBAf0(0, 0, 0, 0.2), strokevisible=false
+            ) 
+            facetlayout[1, i, Top()] = text
+        end
+    
+        # Shared xlabel
+        group_bottom_protrusion = lift(
+            (xs...) -> maximum(y -> y.bottom, xs),
+            (MakieLayout.protrusionsobservable(ax) for ax in axs[end, :])...
+        )
+    
+        padx = Node(10.0)
```

Comment:
Do we need this here? Somehow, I imagined that the defaults of MakieLayout should also be our defaults, or is there something MakieLayout is not doing automatically?

Reply:
@jkrumbiegel, would you mind helping us out? I basically copied your code from slack :-D 

Are similar the paddings between regular axis labels and tick labels similarly hard coded in MakieLayout? Or could extract the padding from the given plot and and use it here?

Reply:
well you could of course just use one of the padding attributes of one of the LAxis instances, it's the same thing but if you have a spanning label, this parameter doesn't belong to one of the spanned axes... I just wanted to show in the example how you can correctly position the label, it's your decision how you want to expose that parameter :)

Reply:
thanks!

Reply:
By default, the padding should be identical for all panels, right? (The axis labels are always nicely aligned). That is, I can just take the attribute from any panel.

Reply:
I'm not sure if I understand what you want to know. Normally, an axis has an x and y label and there is a padding value for those against the tick labels. What you're doing here is not using those labels at all, but simply sticking another label across the outer axes and padding it so it looks good. The extra padding value that you need can come from anywhere, but why would you reuse it from one of the axes? Then the user would have to go to that axis and change the padding value there. I think it makes most sense to make these values keyword arguments of the AoG function that creates the grid. I don't know if the AoG data structure in the end has a space for storing this attribute, though.

Reply:
Thanks, I think I misunderstood which distance the padding measures.